### DISCUSSION TITLE
install on any cpanel

### DISCUSSION THREAD

**Discussion by tyuhbh0** [Nikeev/sesdashboard#59] [General]
is it possible to install sesDashboard on any panel such as, plesk, cpanel pr aapanel? now?

**Comment by Nikeev** [comment]
Hello! I didn't test it with plesk, cpanel or aapanel. But if your server supports PHP and MySQL, I think you could install it with any panel according to https://sesdashboard.readthedocs.io/en/latest/install.html#shared-hosting-installation

**Comment by tyuhbh0** [comment]
after unzip in root folder, anything should to install sesdashboard? or that is it to login?

is it works for subdomain?  such as, https://subdomain.domain.com

what is installation url?
